python/cudf/cudf/_lib/groupby.pyx

```diff
@@ -254,3 +271,35 @@ def _drop_unsupported_aggs(Table values, aggs):
         raise DataError("No numeric types to aggregate")
```

Comment:
@shwina 
in `_drop_unsupported_aggs`,
```python
            for i, agg_name in enumerate(aggs[col_name]):
                if Aggregation(agg_name).kind not in _DECIMAL_AGGS:
                    del result[col_name][i]
```
This logic may cause `IndexError: list assignment index out of range`.